static/js/util.ts

```diff
@@ -206,28 +217,28 @@ export function sorted_ids(ids) {
     return id_list;
 }
 
-export function set_match_data(target, source) {
+export function set_match_data(target: Message, source: Message): void {
     target.match_subject = source.match_subject;
     target.match_content = source.match_content;
 }
 
-export function get_match_topic(obj) {
+export function get_match_topic(obj: Message): string | undefined {
     return obj.match_subject;
 }
 
-export function get_draft_topic(obj) {
+export function get_draft_topic(obj: Message): string {
     // We will need to support subject for old drafts.
     return obj.topic || obj.subject || "";
 }
 
-export function get_reload_topic(obj) {
+export function get_reload_topic(obj: Message): string {
     // When we first upgrade to releases that have
     // topic=foo in the code, the user's reload URL
     // may still have subject=foo from the prior version.
     return obj.topic || obj.subject || "";
 }
 
-export function get_edit_event_topic(obj) {
+export function get_edit_event_topic(obj: Message): string | undefined {
```

Comment:
This is not called on a message object, but rather an [`update_message` event object](https://zulip.com/api/get-events#update_message).